front/src/applications/operationalStudies/components/ManageTrainSchedule/Itinerary/DisplayItinerary/Origin.tsx

```diff
@@ -49,31 +46,18 @@ function Origin(props: OriginProps) {
   const originLinkedBounds = useSelector(getOriginLinkedBounds);
   const originUpperBoundDate = useSelector(getOriginUpperBoundDate);
   const originUpperBoundTime = useSelector(getOriginUpperBoundTime);
-  const pathfindingID = useSelector(getPathfindingID);
   const dispatch = useDispatch();
   const { t } = useTranslation(['operationalStudies/manageTrainSchedule']);
-  const { openModal } = useContext(ModalContext);
 
   const { isByOrigin, isByDestination } = makeEnumBooleans(STDCM_MODES, stdcmMode);
   const { isSimulation, isStdcm } = makeEnumBooleans(MODES, mode);
 
-  const openModalWrapperBecauseTypescriptSucks = () => {
-    openModal(<ModalPathJSONDetail />, 'lg');
-  };
-
   const originTitle = (
     <h2 className="d-flex align-items-center mb-0 pl-4">
       <span className="mr-1 h2 text-success">
         <RiMapPin2Fill />
       </span>
       <span>{t('origin')}</span>
-      <button
-        type="button"
-        onClick={openModalWrapperBecauseTypescriptSucks}
-        className="btn btn-link"
-      >
-        <small className="ml-1">{pathfindingID}</small>
-      </button>
```

Comment:
Okay it was not the better name :D But the functionnality is needed, it gives the possibility to export json pathFinding data for research purpose, I will explain to you on Slack. Please always ask before completely remove a functionality, thx :)